src/pydargs/__init__.py

```diff
@@ -33,47 +33,47 @@ class ADataclass(Protocol):
 Dataclass = TypeVar("Dataclass", bound=ADataclass)
 
 
-def parse(tp: Type[Dataclass], args: Optional[list[str]] = None) -> Dataclass:
+def parse(tp: Type[Dataclass], args: Optional[list[str]] = None, **kwargs: Any) -> Dataclass:
     if args is None:
-        args = sys.argv
-    namespace = _create_parser(tp).parse_args(args)
+        args = sys.argv[1:]
```

Comment:
This was part of #43 
```suggestion
        args = sys.argv
```

Reply:
Yes my plan was to rebase after that got merged, then this will disappear 👍